xds/src/test/java/io/grpc/xds/ServerXdsClientTest.java

```diff
@@ -759,6 +759,15 @@ public void streamClosedAndRetry() {
         backoffPolicy2);
   }
 
+  @Test
+  public void getSubscribedResources() {
```

Comment:
Such a test is fairly weak. It would be much more robust to verify xDS requests sent to an in-process service implementation.